Feliz/React.fs

```diff
@@ -62,16 +62,24 @@ type React =
         { new IDisposable with member _.Dispose() = dispose() }
 
     /// The `React.fragment` component lets you return multiple elements in your `render()` method without creating an additional DOM element.
-    static member inline fragment xs = Fable.React.Helpers.fragment [] xs
+    static member inline fragment xs =
+        Fable.React.ReactBindings.React.createElement(Fable.React.ReactBindings.React.Fragment, obj(), xs)
 
     /// The `React.fragment` component lets you return multiple elements in your `render()` method without creating an additional DOM element.
-    static member inline keyedFragment(key: int, xs) = Fable.React.Helpers.fragment [ !!("key", key) ] xs
+    static member inline keyedFragment(key: int, xs) = // Fable.React.Helpers.fragment [ !!("key", key) ] xs
+        Fable.React.ReactBindings.React.createElement(Fable.React.ReactBindings.React.Fragment, createObj ["key" ==> key], xs)
```

Comment:
No need to convert `xs` to an array here?

Reply:
The children argument of `ReactBindings.React.createElement` is decorated with `ParamList` attribute (alias for `ParamSeq`), this is similar to `ParamArray` but it allows for any `seq` type because in JS you can spread any iterable not only arrays. I added it in Fable 2 specifically for React.createElement so we didn't have to do the runtime array conversion just to use lists in F#.

But if you mean `React.Children.toArray` to avoid the React warning about the keys, I'm not sure, I'd have to check. Although in principle this will generate the same code as before when calling `Fable.React.Helpers.fragment`.